app/models/solidus_paypal_braintree/source.rb

```diff
@@ -79,12 +83,21 @@ def display_number
     end
 
     private
```

Comment:
⚒ Could we add a whitespace line after the `private` ?

```suggestion
    
    private
```